### DISCUSSION TITLE
[Warning] Don't have Chinese starting

### DISCUSSION THREAD

**Discussion by bowuL** [Django-Simple-API/django-simple-api#39] [General]
干啥子不整个中文介绍呢，多亲切啊

**Comment by waitan2018** [comment]
你都没给 star，不整（赶紧 star

**Comment by bowuL** [comment]
Check，please ------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年4月1日(星期四) 中午11:46
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Django-Simple-API/django-simple-api] [Warning] Don&#39;t have Chinese starting (#39)
